profile/README.md

```diff
@@ -1,28 +1,27 @@
 
-
 ![Banner](https://user-images.githubusercontent.com/72812470/134735139-8dd74c7f-82b3-4a80-8e28-205ba04ce345.png)
 <br>
 <p align="center">
-<a href= "https://github.com/web3community/web3community.github.io/issues" alt = "Open-issues">
-  <img src = "https://img.shields.io/github/issues/web3community/web3community.github.io" target="blank" />
+<a href= "https://github.com/WebXDAO/WebXDAO.github.io/issues" alt = "Open-issues">
+  <img src = "https://img.shields.io/github/issues/WebXDAO/WebXDAO.github.io" target="blank" />
 </a>
 <a href="https://discord.gg/TSRwqx4K2v" alt="community-discord-server">
    <img src="https://img.shields.io/discord/835424705410236427?logo=discord&?style=for-the-badge&colorB=5865F2" target="blank" />
 </a>
-<a href= "https://github.com/web3community" alt = "Stars">
-  <img src = "https://img.shields.io/github/stars/web3community?style=social" target="blank" />
+<a href= "https://github.com/WebXDAO" alt = "Stars">
+  <img src = "https://img.shields.io/github/stars/WebXDAO?style=social" target="blank" />
 </a>
-<a href="https://twitter.com/web3community" alt="Follow Web3Community on Twitter">
-<img src="https://img.shields.io/twitter/follow/web3community?label=Web3Community&style=social" /></a>
-<a href="https://github.com/web3community/start-here" alt="License">
-<img src="https://img.shields.io/github/license/web3community/start-here.svg" /></a>
+<a href="https://twitter.com/WebXDAO" alt="Follow WebXDAO on Twitter">
+<img src="https://img.shields.io/twitter/follow/WebXDAO?label=WebXDAO&style=social" /></a>
+<a href="https://github.com/WebXDAO/start-here" alt="License">
+<img src="https://img.shields.io/github/license/WebXDAO/start-here.svg" /></a>
 </p>
 
 <br>
 
-[Web3Community](https://web3community.github.io/) is a community that focus on Web 3, Blockchain, and somewhat related to decentralized applications (Dapp). Here you will learn how to become a blockchain developer while having fun with other community members. Blockchain Developer is an interesting field, in demand and the latest trend in the tech industry. Here you will get to interact with some great folks and learn blockchain technology together. This community is in its initial stage so you can show your leadership skills and help for everyone's success. The community is powered by [Dev Protocol](https://devprotocol.xyz/).
+[WebXDAO](https://WebXDAO.github.io/) is a community that focus on Web 3, Blockchain, and somewhat related to decentralized applications (Dapp). Here you will learn how to become a blockchain developer while having fun with other community members. Blockchain Developer is an interesting field, in demand and the latest trend in the tech industry. Here you will get to interact with some great folks and learn blockchain technology together. This community is in its initial stage so you can show your leadership skills and help for everyone's success. The community is powered by [Dev Protocol](https://devprotocol.xyz/).
```

Comment:
We are just not focusing on these techs but on the future of the web.

Reply:
CC @KukretiShubham @shuklaritvik06

Reply:
> We are just not focusing on these techs but on the future of the web.

+1, We will discuss more on description and banner in our discord server for now in this PR, I think it will be good to update links only 😅